pallets/runtime/common/tests/all/settlement_test.rs

```diff
@@ -0,0 +1 @@
+../../../../settlement/src/tests.rs
```

Comment:
What is the purpose of this file? Is it supposed to be a symbolic link?

Reply:
Yes, this is a symbolic link to the test file in the settlement submodule.

Reply:
Funny that GitHub shows it this way :)